### PR TITLE
Webhook imported from Yahoo Inc

### PR DESCRIPTION
**Pull request by mlajkim**
# Description

Garm relies on Yahoo's repository [k8s-athenz-webhook](https://github.com/yahoo/k8s-athenz-webhook), which is no longer regularly updated. Through this PR, we're bringing in the original code from the repository so we can manage it ourselves from here on out.

## Type of change

- [ ] Bug fix
- [ ] New feature
- [X] Refactoring (no functional changes, no api changes)
- [ ] Non-code changes (update documentation, pipeline, etc.)

### Flags

- [ ] Breaks backward compatibility
- [ ] Requires a documentation update
- [ ] Has untestable code

---

## Checklist

- [ ] Followed the guidelines in the CONTRIBUTING document
- [ ] Added prefix `[skip ci]`/`[ci skip]`/`[no ci]`/`[skip actions]`/`[actions skip]` in the PR title if necessary
- [x] Tested and linted the code
- [ ] Commented the code
- [ ] Made corresponding changes to the documentation
- [x] Passed all pipeline checking

## Checklist for maintainer
- [ ] Use `Squash and merge`
- [ ] Double-confirm the merge message has prefix `[skip ci]`/`[ci skip]`/`[no ci]`/`[skip actions]`/`[actions skip]`
- [ ] Delete the branch after merge

### CODE REVIEW

**Comment by mlajkim** [comment]
image

File: third_party/webhook/doc.go [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Package log is originated from https://github.com/yahoo/k8s-athenz-webhook/releases/tag/v0.1.4
```
Comment (WindzCUHK) [inline third_party/webhook/doc.go:1]:
```suggestion
// Package webhook is originated from https://github.com/yahoo/k8s-athenz-webhook/releases/tag/v0.1.4
// Copyright 2019 Verizon Media Inc. Licensed under the terms of the 3-Clause BSD License.
// See LICENSE file in github.com/yahoo/k8s-athenz-webhook/blob/v0.1.4/LICENSE for terms.
```
I think we can just add the terms in here and then remove the files in `yahoo_inc_originated_documents`.
Or, do you have other intentions for keeping the original documents too?

Comment (mlajkim) [reply third_party/webhook/doc.go:1]: @WindzCUHK If it is someone else's work, I think its a good move to preserve what they have written in an original format. Don't want to alter single word from 3rd party.

And Thank you for the suggestion!

Comment (mlajkim) [reply third_party/webhook/doc.go]: (And fixed typo)

**Review by WindzCUHK — APPROVED** [review]

**Pull request merged**